src/main/java/com/sergiofah/service/CategoryService.java

```diff
@@ -0,0 +1,40 @@
+package com.sergiofah.service;
+
+import com.sergiofah.dto.CategoryDTO;
+import com.sergiofah.model.Category;
+import com.sergiofah.repository.CategoryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CategoryService {
+    @Autowired
+    private CategoryRepository categoryRepository;
+
+    public List<CategoryDTO> getCategories(Long id){
```

Comment:
The left brace must always be separated from the previous character by most Java code conventions and our convention.